api-test.sh

```diff
@@ -386,12 +386,18 @@ has_key() {
 
 check_eq() {
   tput cuf 6
-  local type=$(jq -r -c --argjson a "$1" -n '$a|type')
+  local type=$(jq -r -c --argjson a "$1" -n '$a|type' 2&>/dev/null)
```

Comment:
```suggestion
  local type=$(jq -r -c --argjson a "$1" -n '$a|type' 2>/dev/null)
```
In bash when `&` is used the variable is always empty